sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -170,6 +170,24 @@ struct {
 }
 ```
 </td>
+</tr><tr>
+<td>buffer</td>
+<td>
+
+``` C++
+void *

```

Comment:
If the context has multiple devices in it, there might be multiple device allocation backing a SYCL buffer. Which one shall that return? I suggest that instead of letting user to choose a specific Device for which an allocation is wanted (as it doesn't fit to the `get_native()` interface) we document that *any* allocation is returned that has all the data bits representing the contents of the buffer valid. Users can further use `zeMemGetAllocProperties` to know the device of the L0 allocation

Reply:
I thought @againull was proposing to remove support for buffer `get_native` from this PR.  If so, then this question is no longer relevant.

If we do want to support `get_native`, then I think this PR should be updated to return a vector of native handles for the buffer.  That is what the OpenCL backend interop API does.

Reply:
I guess it is not used with the latest changes, right? Can we tell so in the table, i.e. that it is not defined?

Reply:
It is used for get_native_mem() of the interop_handler: https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/interop_handle.hpp#L53-L54
which is supported for L0, that's why I decided to not mark it as not defined.

Reply:
OK